xml/System.Runtime.InteropServices/MemoryMarshal.xml

```diff
@@ -157,12 +185,23 @@
         <Parameter Name="span" Type="System.ReadOnlySpan&lt;TFrom&gt;" />
       </Parameters>
       <Docs>
-        <typeparam name="TFrom">To be added.</typeparam>
-        <typeparam name="TTo">To be added.</typeparam>
-        <param name="span">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TFrom">The type of the source span.</typeparam>
+        <typeparam name="TTo">The type of the target span.</typeparam>
+        <param name="span">The source slice to convert.</param>
+        <summary>Casts a read-only span of one primitive type to a read-only span of another primitive type.</summary>
```

Comment:
I'm concerned about a possible misconception here.  Consider this cast of a scalar:

```csharp
long l = 5;
int i  = (int) l;
```

An int is smaller than a long, and only the lower 4 bytes of the long `l` are assigned to the int `i`.

What happens if we cast a `Span<long>` to a `Span<int>`? I'm guessing there are twice as many `int` elements than `long elements and the memory is re-interpreted. We should clarify that.

Reply:
Yes. The cast always appears to occur in an unchecked context.